api.planx.uk/modules/send/email/index.ts

```diff
@@ -54,12 +55,7 @@ export async function sendToEmail(
 
     // Send the email
     const response = await sendEmail("submit", sendToEmail, config);
-    if (response?.message !== "Success") {
```

Comment:
Again, this code was unreachable - `message` is only set to `"Success"` and nothing else.